sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -573,7 +573,8 @@ class Dataset[T] private[sql](
     require(!IntervalUtils.isNegative(parsedDelay),
       s"delay threshold ($delayThreshold) should not be negative.")
     EliminateEventTimeWatermark(
-      EventTimeWatermark(UnresolvedAttribute(eventTime), parsedDelay, logicalPlan))
+      EventTimeWatermark(util.UUID.randomUUID(), UnresolvedAttribute(eventTime),
```

Comment:
Could you import properly like the other files?
```java
import java.util.UUID
```

Reply:
Ah, nice finding!

Reply:
@dongjoon-hyun
Ah I just remembered. The file has various reference of `util.XXX` already (I guess we tend to use jutil to differentiate with scala.util but we didn't for this file), so if I import the class with full qualifier, it'll be diverged with existing style in the file. Would we like to be consistent with existing file, or across files?